tests/test_jointcal_cfht_minimal.py

```diff
@@ -55,7 +55,7 @@ def test_jointcalTask_2_visits_photometry(self):
         # NOTE: ndof==1 from 4 fit parameters (2 model, 2 fittedStar), and
         # 5 degrees-of-freedom (3 star measurements, with 2 reference stars).
         metrics = {'collected_photometry_refStars': 183,
-                   'selected_photometry_refStars': 183,
+                   'selected_photometry_refStars': 2,
```

Comment:
Yikes! Surely this suggests something is wrong.

Reply:
This is intended: the test is a minimal working example with a total of 3 stars across 2 ccds, designed so that one can do the fitting math "by hand."